recipes/psychopy/meta.yaml

```
@@ -0,0 +1,108 @@
+{% set name = "psychopy" %}
+{% set pypi_name = "PsychoPy" %}
+{% set version = "3.1.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ pypi_name[0] }}/{{ pypi_name }}/{{ pypi_name }}-{{ version }}.zip
+  sha256: 5df7c0ff494368fcc4ade820d5b83135f82a44740868ed199d1023e181c30f95
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install --no-deps . -vv"
```

Assuming this is pure Python:
```suggestion
  noarch: python
  script: "{{ PYTHON }} -m pip install --no-deps . -vv"
```

---

Just noticed there are platform specific dependencies so this can't be `noarch`. In that case you need to change how you constrain the Python version:
```suggestion
  skip: true  # [py<36]
  script: "{{ PYTHON }} -m pip install --no-deps . -vv"
```
and replace `python >=3.6` with `python` ins `host` and `run`.

---

(The use of `>=3.6` without `noarch` is causing the Windows failure you noted above)

---

Thanks, trying that now!